### PR TITLE
fixing syntax errors in nosetests

### PR DESCRIPTION
**Pull request by alvinzz**
--Tests can be run locally with similar output to travis-ci by using "nosetests --exe".
Added coverage module.
Created the music/features/tests/data directory (currently used in tests).
Added the "Beethoven....mp3" test file, and dummy "test[...].wav" files (for tests).
-- These will not be used in future tests, since they compare to another library.
Fixed syntactical errors raised by nosetests.
Added ffmpeg install for travis-ci .mp3 processing.
Fixed inputs of calls to "extractor.rms" in the test suite to correctly match the function.

### CODE REVIEW

**Comment by stefanv** [comment]
Can you please rebase these changes on master?

**Comment by MichaelTamaki** [comment]
@stefanv Can you review this pull request? We think that this is ready to merge.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+python:
+  - "3.5"
+
+#command to install dependencies
+install:
+  - sudo apt-get install ffmpeg
+  - "pip install -r requirements.txt"
+
+#command to run tests
+script: nosetests --exe --with-coverage --cover-package=music-features
```
Comment (MichaelTamaki) [inline .travis.yml]: Maybe change `--cover-package=music_feats`?

Comment (alvinzz) [inline .travis.yml]: Fixed, thanks!

**Pull request merged**